spec/index.html

```diff
@@ -1553,7 +1555,11 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
               such that the <a data-cite="INFRA#map-key">key</a> in <var>e<sub>a</sub></var>
               equals the <a data-cite="INFRA#map-key">key</a> in <var>e<sub>b</sub></var>,
               and the <a data-cite="INFRA#map-value">value</a> in <var>e<sub>a</sub></var>
-              equals the <a data-cite="INFRA#map-value">value</a> in <var>e<sub>b</sub></var>.</li>
+              equals the <a data-cite="INFRA#map-value">value</a> in <var>e<sub>b</sub></var>.
+              <div class="note">Two JSON Objects containing maps which are serialized with entries in a different order will be equal under this definition when transformed to the value space.
+              For example, `{ "a": 1, "b": 2} and {"b":2, "a":1}` are considered equal.
```

Comment:
I wonder if the spaces following the colons in the first serialization, but not in the second, will lead to questions? It may be worth highlighting that these spaces are syntactically and semantically meaningless, or making the map entries actually the same.

Reply:
While I don’t think we need to describe JSON syntax, we should strive for consistency.

Reply:
I don't have a "mark this resolved" button... so here are all the missing spaces, inserted. 
```suggestion
              For example, `{ "a": 1, "b": 2 } and { "b": 2, "a": 1 }` are considered equal.
```